Ansible/ansible_collections/jfrog/platform/roles/artifactory_nginx/tasks/RedHat-ownstream.yml

```diff
@@ -0,0 +1,21 @@
+- name: Enable nginx from AppStream
+  become: true
+  ansible.builtin.command: "yum module enable -y nginx:{{ nginx_module }}"
```

Comment:
I'm trying see if we can  have nginx_module (package version) as `1.27` in redhat and debain based systems as default  . WDYT ?

Reply:
Hmm, I see it can't be latest unfortunately